### ISSUE TITLE
gnome-control-center crashes after going to the privacy tab on bluefin-nvidia-laptop-userns-hardened.

### ISSUE THREAD

**Issue by CornPopGuy** [secureblue/secureblue#152]
Output of gnome control center when ran from terminal on secureblue bluefin nvidia laptop 
```
● ostree-image-signed:docker://ghcr.io/secureblue/bluefin-nvidia-laptop-userns-hardened:latest
                   Digest: sha256:83b54d4b320e2e1608fc95be2c0a457944918c4be1cea972e5522ed8429a69a6
                  Version: 39.20240105.0 (2024-01-05T18:29:06Z)

```
Output from terminal:
```

❯ gnome-control-center
20:06:40.8995                  Gtk[10709]: WARNING: Unknown key gtk-modules in /home/redacted/.config/gtk-4.0/settings.ini
20:06:41.5044                 GLib[10709]:CRITICAL: g_atomic_ref_count_dec: assertion 'old_value > 0' failed
20:06:41.5064     cc-privacy-panel[10709]: WARNING: Failed fetch permissions from flatpak permission store: GDBus.Error:org.freedesktop.portal.Error.NotFound: No entry for location
20:06:41.5064     cc-privacy-panel[10709]: WARNING: Failed fetch permissions from flatpak permission store: GDBus.Error:org.freedesktop.portal.Error.NotFound: No entry for camera
20:06:41.5064     cc-privacy-panel[10709]: WARNING: Failed fetch permissions from flatpak permission store: GDBus.Error:org.freedesktop.portal.Error.NotFound: No entry for microphone
fatal allocator error: detected write after free

```

Notes:
- The control center app worked without any issues on secureblue silverblue.
- I have all gnome extensions disabled.

**Comment by RoyalOughtness** [comment]
This is caused by bluefin pulling in a custom and old version of gnome-control-center. The fix for this issue is in 45.2, but bluefin is on 45.1:

https://github.com/ublue-os/bluefin/blob/main/Containerfile#L23
https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/2647
https://gitlab.gnome.org/GNOME/gnome-control-center/-/merge_requests/2012

I'm cutting an issue to upstream bluefin for this.

**Comment by RoyalOughtness** [comment]
Fixed upstream and ingested here. Please reopen if you can repro.
